pkg/e2e-test.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+#
+# Copyright 2021 The Sigstore Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+testdir=$(dirname "$0")
+
+rm -f /tmp/pkg-rekor-*.cov
+echo "installing gocovmerge"
+make gocovmerge
+docker kill $(docker ps -q) || true
+echo "starting services"
+docker-compose -f docker-compose.yml -f docker-compose.test.yml up -d --force-recreate --build
```

Comment:
do we need `--force-recreate` here?

Reply:
I reused most of them from here https://github.com/sigstore/rekor/blob/7807d1e5a3fbc9b8fdd17053da8163463e87a5ae/tests/e2e-test.sh#L26

which had the reference. I would suggest in my next PR, I can check if it is required and remove it. Would that work?

Reply:
I would vote fix pkg/e2e-test.sh here since you're adding a new file. we can remove from test/e2e-test.sh in another PR.

Reply:
thanks, otherwise we rebuild the same container (which can take a while in CI) for no reason.

Reply:
Fixed it. Thanks